### PR TITLE
daemon: add page

### PR DESCRIPTION
**Pull request by CleanMachine1**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages/linux/daemon.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+- Run a command as a daemon, writing logs to a specific file:
+
+`daemon --name="{{name}}" --errlog={{path/to/logfile}} {{command}}`
+
+- Stop a daemon:
+
+`daemon --name="{{name}}" --stop`
+
+- List daemons:
```
Comment (navarroaxel) [inline pages/linux/daemon.md:26]: An stopped daemon is listed here?

File: pages/linux/daemon.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`daemon --name="{{name}}" --respawn {{command}}`
+
+- Run a command as a daemon which will restart if it crashes, with two attempts every 10 seconds:
+
+`daemon --name="{{name}}" --respawn --attempts=2 --delay=10 {{command}}`
+
+- Run a command as a daemon, writing logs to a specific file:
+
+`daemon --name="{{name}}" --errlog={{path/to/logfile}} {{command}}`
```
Comment (navarroaxel) [inline pages/linux/daemon.md:20]:
```suggestion
`daemon --name="{{name}}" --errlog={{path/to/file.log}} {{command}}`
```

File: pages/linux/daemon.md (hunk 1 above)
Comment (CleanMachine1) [reply pages/linux/daemon.md:26]: I think I should be more clear, the stop here sends a SIGTERM

File: pages/linux/daemon.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Run a command as a daemon which will restart if it crashes, with two attempts every 10 seconds:
+
+`daemon --name="{{name}}" --respawn --attempts=2 --delay=10 {{command}}`
+
+- Run a command as a daemon, writing logs to a specific file:
+
+`daemon --name="{{name}}" --errlog={{path/to/file.log}} {{command}}`
+
+- Stop a daemon:
```
Comment (CleanMachine1) [inline pages/linux/daemon.md:22]:
```suggestion
- Kill a daemon:
```

File: pages/linux/daemon.md (hunk 1 above)
Comment (mfrw) [reply pages/linux/daemon.md:26]: Opinion: Should we add a `(SIGTERM)` in bracket for the stop/kill example ?

Comment (CleanMachine1) [reply pages/linux/daemon.md:26]: I see no reason not to @mfrw

File: pages/linux/daemon.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+- Run a command as a daemon which will restart if it crashes, with two attempts every 10 seconds:
+
+`daemon --name="{{name}}" --respawn --attempts=2 --delay=10 {{command}}`
+
+- Run a command as a daemon, writing logs to a specific file:
+
+`daemon --name="{{name}}" --errlog={{path/to/file.log}} {{command}}`
+
+- Kill a daemon:
```
Comment (CleanMachine1) [inline pages/linux/daemon.md:22]:
```suggestion
- Kill a daemon (SIGTERM):
```

**Review by blueskyson — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**